client/app/main/student/student.controller.js

```diff
@@ -2,18 +2,8 @@
 
 var app = angular.module('app');
 
-function StudentCtrl($scope, $stateParams, Intervention, Modal, Outreach,
-  Student, toastr) {
-
+function StudentCtrl($scope, $state, $stateParams, Student, toastr) {
   $scope.student = Student.get({id: $stateParams.id});
```

Comment:
Gotcha. The bug Jennifer mentioned was only related to the Outreach Actions, should I add expanded note view to Interventions as well? Maybe just a different ticket?